src/client.js

```diff
@@ -58,20 +58,14 @@ class StitchClient {
   }
 
   /**
-   * Login to stich instance, optionally providing a username and password. In
-   * the event that these are omitted, anonymous authentication is used.
-   *
-   * @param {String} [email] the email address used for login
-   * @param {String} [password] the password for the provided email address
-   * @param {Object} [options] additional authentication options
-   * @returns {Promise}
-   */
+  * Convenience method for authenticating with the user/password auth provider
```

Comment:
This comment isn't entirely accurate - it also falls back to anonymous auth, if either username or password isn't passed.

Reply:
Oversight, originally I had this function entirely removed before realizing the implications of removing a client-level method.  Going to add it back and re-word accordingly.